week-2-practise/level-2/6-map.js

```diff
@@ -2,13 +2,21 @@
 // 1. Write a function that formats the numbers into a string with the percentage symbol appended e.g. "10%"
 // 2. Declare a new array, `percentagesFormatted`, containing
 //    - each item in `percentages` formatted by your function
-
+function percentage(number){
+  return number + "%"
+}
 var percentages = [1, 23, 92, 18];
-
+var percentagesFormatted = percentages.map(percentage)
 console.log(percentagesFormatted);
 
 /* 
   EXPECTED RESULT
   ---------------
   [1%, 23%, 92%, 18%]
 */
+function double(num) {
+  return num * 2;
+}
+
+var numbers = [1, 2, 3, 4];
+var numbersDoubled = numbers.map(double);
```

Comment:
Following the comment I made above, the code here would be
```
var doubles = numbers.map(function(num) {
  return double(num);
});
```

This is like saying "take the array 'numbers', go over each item inside it and perform the action described in the function 'double(num)' on each item".

Reply:
Thanks